db-ops/job-create-db-mysql.yml.tpl

```diff
@@ -2,51 +2,45 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: db-ops
+
 ---
 apiVersion: v1
 kind: Secret
 metadata:
   name: db-create-users
   namespace: db-ops
 type: Opaque
-stringData: 
+stringData:
+  RDS_MASTER_PASSWORD: $MASTER_RDS_PASSWORD
   create-user.sql: |
-    DROP USER '$DB_APP_USERNAME';
+    CREATE DATABASE IF NOT EXISTS $DB_NAME;
```

Comment:
Creation would fail without these

Reply:
the terraform RDS module we use creates a database so it should be fine for most cases

Reply:
Yeah though if there is a dash in the name I'm removing it which I think means it might not exist.

Reply:
yeah we have a line in the database module to do that too https://github.com/commitdev/terraform-aws-zero/blob/main/modules/database/main.tf#L61